### DISCUSSION TITLE
Nginx support

### DISCUSSION THREAD

**Discussion by aidendotgg** [coollabsio/coolify#4097] [Feature Requests]
I have been loving coolify but trying to config traefik and caddy for certain things has been very tedious. I have an existing nginx config from before using coolify that I know for a fact will work and if I was able to use nginx, coolify would be an 11/10 for me. Please consider adding it 🙏🏻

**Comment by elliotjarnit** [comment]
Agreed. Nginx is the industry standard and there has been talk about implementing it in the past. Any update on its progress?

**Comment by familiar** [comment]
^^ lf updates
